src/class/Car.js

```diff
@@ -0,0 +1,24 @@
+export default class Car {
+  name;
+  distance;
+
+  constructor(name, advanceCondition) {
+    this.name = name;
+    this.distance = 0;
+    this.advanceCondition = advanceCondition;
+  }
+
+  getName() {
+    return this.name;
+  }
+
+  getDistance() {
+    return this.distance;
+  }
```

Comment:
이미 name을 외부에서 접근 가능한 상태라서, getName 같은 메소드를 만드는건 무의미해보여요!
이를 잘 활용하려면 차라리 이렇게 해보면 어떨까요?

```suggestion
export default class Car {
  #name;
  #distance;

  constructor(name, advanceCondition) {
    this.#name = name;
    this.#distance = 0;
    this.advanceCondition = advanceCondition;
  }

  get name() {
    return this.#name;
  }

  get distance() {
    return this.#distance;
  }
```

이렇게 private 접근제한자를 지정하고, getter로 표현해주면 어떨까 싶네요!